src/utils/string.ts

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { split } from "lodash";
+import { split } from "lodash-es";
```

Comment:
Also worth pointing out EW hit bugs in Lodash's split grapheme splitting, we had to switch to `graphemer`

Reply:
See https://github.com/matrix-org/matrix-react-sdk/pull/10976

Reply:
https://github.com/beaugunderson/emoji-aware/issues/25 & https://github.com/lodash/lodash/issues/5652 being related issues about this bug

Reply:
Have opened https://github.com/vector-im/compound/issues/196 to track this